pkg/users/group-details.js

```diff
@@ -0,0 +1,213 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2023 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import cockpit from 'cockpit';
+import React, { useState } from 'react';
+import { superuser } from "superuser";
+import { apply_modal_dialog } from "cockpit-components-dialog.jsx";
+import { rename_group_dialog } from "./rename-group-dialog.js";
+import { delete_group_dialog } from "./delete-group-dialog.js";
+import { show_unexpected_error } from "./dialog-utils.js";
+
+import {
+    Button,
+    Card, CardBody, CardHeader, CardTitle, CardActions,
+    Dropdown, DropdownItem, KebabToggle,
+    Flex, HelperText, HelperTextItem,
+    Label, LabelGroup,
+    Page, PageSection,
+    Select, SelectOption, SelectVariant, Text, TextVariants, Breadcrumb, BreadcrumbItem,
+    Form, FormGroup,
+} from '@patternfly/react-core';
+import { UndoIcon } from '@patternfly/react-icons';
+
+const _ = cockpit.gettext;
+
+const GroupActions = ({ group, groups, accounts }) => {
+    const [isKebabOpen, setKebabOpen] = useState(false);
+
+    const actions = [
+        <DropdownItem key="rename-group"
+                      onClick={ev => { setKebabOpen(false); rename_group_dialog(group) }}>
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18136-20230110-115950-e62ee64c-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)